eng/pipelines/vmr-sync-internal.yml

```diff
@@ -5,6 +5,10 @@ trigger:
   branches:
     include:
     - internal/release/*
+    exclude:
+    - internal/release/8.0.2xx
```

Comment:
Thought on utilizing a wild card for 8 to avoid maintenance for each major release?

internal/release/*.0.2xx 
or 
internal/release/?.0.2xx and internal/release/??.0.2xx

Reply:
I added the wildcard but I don't think it really matters, 9.0 will be not be synchronized from here but through the new service.